packages/holocron-dev-server/src/utils/helpers.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import open from 'open';
+
+export function isDevelopment() {
+  return process.env.NODE_ENV !== 'production';
+}
+
+export function getLocalRootModule({ modules }) {
+  return modules.find(({ rootModule }) => rootModule);
+}
+
+export function getModuleFromFilePath({ modules, filePath }) {
+  return modules.find(({ modulePath }) => filePath.startsWith(modulePath));
+}
+
+export function getModuleInfoFromLocalePath(filePath) {
+  const [modulePath, fileBasePath = ''] = filePath.split('/locale/');
+  const [moduleName] = modulePath.split('/').reverse();
+  const [languagePack] = fileBasePath.split('/');
+  const locale = languagePack.replace('.json', '');
+  return [moduleName, modulePath, locale, languagePack];
+}
+
+export function openBrowser(Url) {
+  return open(Url);
+}
```

Comment:
why is this needed?

Reply:
This allows users to set a url that can be launched when hmr startups - it one of the configurable options otherwise it will open localhost:4000